libr/core/project.c

```diff
@@ -521,7 +521,7 @@ static bool store_files_and_maps (RCore *core, RIODesc *desc, ut32 id) {
 	RListIter *iter;
 	RIOMap *map;
 	if (desc) {
-		r_cons_printf ("ofs %s %s\n", desc->uri, r_str_rwx_i (desc->perm));
+		r_cons_printf ("ofs \"%s\" %s\n", desc->uri, r_str_rwx_i (desc->perm));
```

Comment:
The whole command should be quoted instead but that wont work because you need to pass a second argument